src/main/java/io/jenkins/plugins/audit/config/AuditLogConfiguration.java

```diff
@@ -0,0 +1,51 @@
+package io.jenkins.plugins.audit.config;
+
+import hudson.Extension;
+import jenkins.model.GlobalConfiguration;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import java.util.Objects;
+
+@Extension
+public class AuditLogConfiguration extends GlobalConfiguration {
+    private String logDestination;
+
+    public AuditLogConfiguration() {
+        load();
```

Comment:
Should add a call to `reloadLogger()` here as well to ensure the config is updated on startup.